README.md

```diff
@@ -128,4 +128,315 @@ python ss.py -w -is -v ~/Desktop/ff2-screenrecord-first4min.mov
+found text ocr----
+Crew: Coptoin Ceci We ore obout t0 orrive! cecil:Good. 1 ,
+----
+Time taken: 0.26425909996032715 seconds
+finished ocr - Crew: Coptoin Ceci We ore obout t0 orrive! cecil:Good. 1 , 
+eval_data\images\FF2_EN_2.jpg
+eval_data\images\FF2_EN_3.jpg
+eval_data\images\FF2_EN_4.jpg
+eval_data\images\FF2_EN_5.jpg
+<PIL.Image.Image image mode=RGB size=879x601 at 0x200FF501580>
+Images differ by 20
+Images are more than 7 hamming distance. Proceed with OCR
+INFO:root:OCR found text: Crew: Why are We robbing crystols from nnocent People? Crew: Thot' s our duty.
+INFO:root:Time taken: 0.15877485275268555 seconds
+found text ocr----
+Crew: Why are We robbing crystols from nnocent People? Crew: Thot' s our duty.
+----
+Time taken: 0.15877485275268555 seconds
+finished ocr - Crew: Why are We robbing crystols from nnocent People? Crew: Thot' s our duty. 
+eval_data\images\FF2_EN_6.jpg
+<PIL.Image.Image image mode=RGB size=879x601 at 0x200FF28E120>
+Images differ by 33
+Images are more than 7 hamming distance. Proceed with OCR
+INFO:root:OCR found text: Crew:do We red Iy have t0 keep doing this?
+INFO:root:Time taken: 5.086591005325317 seconds
+found text ocr----
+Crew:do We red Iy have t0 keep doing this?
+----
+Time taken: 5.136143445968628 seconds
+finished ocr - Crew:do We red Iy have t0 keep doing this? 
+eval_data\images\FF2_EN_7.jpg
+<PIL.Image.Image image mode=RGB size=879x601 at 0x200FF8010A0>
+Images differ by 0
+Difference is less than 10%. No need to call OCR again.
+eval_data\images\FF2_EN_8.jpg
+eval_data\images\FF2_EN_9.jpg
+eval_data\images\FF2_EN_10.jpg
+=========== Evaluating has text detector ==========
+Precision : 1.0, Recall : 0.4, F1 : 0.5714285714285715, Accuracy : 0.4
+Incorrect filenames ['FF2_EN_10', 'FF2_EN_2', 'FF2_EN_3', 'FF2_EN_4', 'FF2_EN_8', 'FF2_EN_9']
+==================== Evaluating Detection Performance ======================
+{'recall': 0.3612068965517242,
+ 'precision': 0.35204059324045833,
+ 'hmean': 0.35656484450205095,
+ 'IoU': 0.2342075581100041}
+==================== Evaluating Recognition Performance ======================
+Average Character Error Rate (CER): 13.09%
+Average Word Error Rate (WER): 23.53%
+
+Wrong Characters:
+Filename: FF2_EN_1, Predicted: o, Ground Truth: a
+Filename: FF2_EN_1, Predicted: o, Ground Truth: a
+Filename: FF2_EN_1, Predicted: , Ground Truth: l,
+Filename: FF2_EN_1, Predicted: o, Ground Truth: a
+Filename: FF2_EN_1, Predicted: o, Ground Truth: a
+Filename: FF2_EN_1, Predicted: 0, Ground Truth: o
+Filename: FF2_EN_1, Predicted: o, Ground Truth: a
+Filename: FF2_EN_1, Predicted: c, Ground Truth: C
+Filename: FF2_EN_1, Predicted: . 1 , Ground Truth:
+Filename: FF2_EN_5, Predicted: o, Ground Truth: a
+Filename: FF2_EN_5, Predicted: , Ground Truth: i
+Filename: FF2_EN_5, Predicted: o, Ground Truth: a
+Filename: FF2_EN_5, Predicted:  , Ground Truth:
+Filename: FF2_EN_6, Predicted: d I, Ground Truth: all
+Filename: FF2_EN_6, Predicted: 0, Ground Truth: o
```

Comment:
Do we really need all this stuff in the README.md?

Reply:
I've removed these.